pkg/minikube/machine/cache_images.go

```diff
@@ -98,6 +97,27 @@ func LoadImages(cmd bootstrapper.CommandRunner, images []string, cacheDir string
 	return nil
 }
 
+// DeleteImages deletes images from local daemon
```

Comment:
Whats the use case for deleting them from the minikube daemon?  I think the delete function should just clean up the cache folder on the host if anything.

Reply:
my bad, i thought it was meant to delete from the daemon as well. sounds good!